aries-site/src/examples/components/pagination/PaginationTableExample.js

```diff
@@ -487,13 +487,15 @@ const columns = [
       </Box>
     ),
     render: datum => (
-      <Box pad={{ vertical: 'xsmall' }}>
+      <Box direction="row" gap="xxsmall" pad={{ vertical: 'xsmall' }}>
         <Meter
+          alignSelf="center"
           values={[{ value: datum.pinned / datum.pinnable, color: 'graph-2' }]}
           max={1}
           thickness="small"
           size="small"
         />
+        <Text>{((datum.pinned / datum.pinnable) * 10).toFixed(0)}%</Text>
```

Comment:
```suggestion
        <Text>{((datum.pinned / datum.pinnable) * 100).toFixed(0)}%</Text>
```
Think it should be * 100, this looks like "80%"
<img width="221" alt="Screen Shot 2021-05-06 at 4 13 32 PM" src="https://user-images.githubusercontent.com/12522275/117376484-15d40800-ae86-11eb-8350-9aa7af9d5758.png">

Reply:
If so we should adjust in the other examples also